scripts/pre-commit-formatR.R

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env Rscript
+
+if (!require("pacman", quietly = TRUE)) install.packages("pacman")
```

Comment:
I think installing without telling the user is a bit impolite. Why not just fail? The use CRAN install that package manually, no?

Reply:
As I said above, the idea of pre-commit is that it is set and forget, so that was the idea behind this part: better to make it work than have it fail. It can be removed if you want to go in the other direction though.

Reply:
I see where you are coming from, but I think most people think the design philosophy of {pacman} is questionable for usage in packaged code, but that's just my opinion.